.pre-commit-config.yaml

```diff
@@ -18,7 +18,6 @@ repos:
     hooks:
       - id: shellcheck
         types: []
-        args: ["-o", "all", "-e", "2148", "-e", "SC2033", "-e", "SC2032", "-e", "SC2034", "-e", "SC2154", "-e", "SC2312", "-e", "SC2164", "-e", "SC2143", "-e", "SC2103"]
```

Comment:
why are only some of the exceptions included in the new file?

Reply:
Read the comment

Reply:
It doesn't explain why you removed half of them

Reply:
> 3 of the 4 codes I re-enabled triggered no warnings when run on all files, the fourth - only for one file, which I fixed.

Disabling those codes is not relevant anymore

Reply:
why?

Reply:
Because they trigger no warnings when run on all pacscripts, therefore, whatever the reason for disabling them was, there isn't one anymore

Reply:
Future proofing? Reading them, they're clearly not unimportant:
https://www.shellcheck.net/wiki/SC2148

Reply:
https://www.shellcheck.net/wiki/SC2033
These are shellchecks that are disabled so they don't fail on pacscripts and how they're structured.

Reply:
https://www.shellcheck.net/wiki/SC2143
This one is a styling choice.

Reply:
As is https://www.shellcheck.net/wiki/SC2103

Reply:
Ok so, SC2148 is made irrelevant by explicitly specifying shell=bash, because all pacscripts are in bash. SC2103 is irrelevant because we had to use `cd ..` a lot before, but don't anymore. The other two I just think are fine to re-enable, considering there's no good reason for disabling them

Reply:
for 2103, we want to discourage the usage of subshells, so disabling it is important imo

Reply:
Yeah but one can use pushd / popd instead of cd in those cases

Reply:
Just re-disable this one please. I'm fine with ceding the others